pyomo/contrib/mindtpy/mip_solve.py

```diff
@@ -60,20 +58,19 @@ def solve_OA_master(solve_data, config):
         MindtPy.MindtPy_oa_obj = Objective(
             expr=main_objective.expr + MindtPy.MindtPy_penalty_expr,
             sense=main_objective.sense)
-    elif config.add_slack == False:
+    elif config.add_slack is False:
```

Comment:
This is redundant given the test above (and that `add_slack` is in domain bool with a default value).  It would be more clear to implement this as:
```python
else: # not config.add_slack
```